spec/wormbots/world_spec.rb

```diff
@@ -1,11 +1,12 @@
-require 'wormbots'
+require 'wormbots/world'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.